scripts/src/java/org/oppia/android/scripts/proto/coverage.proto

```diff
@@ -33,8 +33,8 @@ message CoveredLine {
 enum Coverage {
   // Coverage status is unspecified.
   UNSPECIFIED = 0;
-  // The line, branch, or function is fully covered, ie. executed atleast once.
+  // The line is fully covered, ie. executed atleast once.
```

Comment:
```suggestion
  // The line is fully covered in a test, that is, it was executed at least once.
```

Some small wording tidying.

Reply:
Thanks, updated the comments as suggested.